src/redux/services/validation.ts

```diff
@@ -75,10 +72,6 @@ function isManagedByKustomize(resource: K8sResource): boolean {
     return true;
   }
 
-  if (resource.refs?.some(hasKustomizationReference)) {
```

Comment:
this check excludes vanilla resources that are used in a kustomization so I removed it - couldn't come up with a scenario where those should be removed!?

Reply:
It gives false positives. I wrote an example in #1695.

Reply:
thanks for clarifying @WitoDelnat - I understand now although I think this scenario is much less common than just including valid resources in your kustomization setup (which seems to be the common case), so I suggest we remove this check for now. I'd rather have the UI show false positives in the uncommon scenario vs false negatives in the common one.

Going forward we we could improve the exclusion logic to exclude resources that are used by kustomizations that contain patches (would have to be recursive) - in which case I also think we should mark in the UI if/why a resource was excluded from validation so users don't incorrectly believe their resources are valid when they actually aren't.

thoughts @devcatalin ? @chlete ?

Reply:
I also agree with @olensmar with following the most common scenario and improve as we move forward.